discovery/discovery_test.go

```diff
@@ -299,31 +273,31 @@ func TestNewDBOrchestratorPoolCache_TestURLs(t *testing.T) {
 		return &net.OrchestratorInfo{
 			Transcoder: "transcoderfromtestserver",
 			PriceInfo: &net.PriceInfo{
-				PricePerUnit:  999,
+				PricePerUnit:  1,
 				PixelsPerUnit: 1,
 			},
 		}, nil
 	}
 
-	dbh, dbraw, err := common.TempDB(t)
-	defer dbh.Close()
-	defer dbraw.Close()
-	require := require.New(t)
-	assert := assert.New(t)
-	require.Nil(err)
+	sender := &pm.MockSender{}
+	node := &core.LivepeerNode{
+		Database: dbh,
+		Eth: &eth.StubClient{
+			Orchestrators: orchestrators,
+		},
+		Sender: sender,
+	}
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
-	node, _ := core.NewLivepeerNode(nil, "", nil)
-	node.Database = dbh
+	sender.On("ValidateTicketParams", mock.Anything).Return(nil).Times(2)
```

Comment:
Why do we need this here? I don't see a call to `GetOrchestrators()`

Reply:
fixed in e8acbd03